core/store/models/job_spec_v2.go

```diff
@@ -0,0 +1,76 @@
+package models
+
+import (
+	"database/sql/driver"
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/lib/pq"
+	"github.com/libp2p/go-libp2p-core/peer"
+	"github.com/pkg/errors"
+)
+
+type (
+	JobSpecV2 struct {
+		ID                            int32 `gorm: "primary_key"`
+		OffchainreportingOracleSpecID int32
+		OffchainreportingOracleSpec   *OffchainReportingOracleSpec `gorm:"save_association:true;association_autoupdate:true;association_autocreate:true"`
+		PipelineSpecID                int32
```

Comment:
Just so I am clear, the pipeline_spec_id foreign key has moved from oracle_spec onto the job itself?

Makes sense.